src/ui/Startup.cs

```diff
@@ -63,7 +123,11 @@ public void ConfigureServices(IServiceCollection services)
                 options.ClientSecret = clientSecret;
                 options.ResponseType = OpenIdConnectResponseType.Code;
                 options.GetClaimsFromUserInfoEndpoint = true;
-                options.UseTokenLifetime = true;
+
+                // using this property would align the expiration of the cookie
+                // with the expiration of the identity token
+                // UseTokenLifetime = true;
```

Comment:
I don't understand from the comment why this is commented out but still present

Reply:
In our case, we want to align it with the lifetime of the access token, so we have to make a couple of changes.

Reply:
comments in the PR will be lost to the mists of time. I suggest the comment in the code be made clearer.

minor.